app/services/customers/update_service.rb

```diff
@@ -146,13 +146,13 @@ def call
         update_result.raise_if_error!
       end
 
-      result.customer = customer
-
       IntegrationCustomers::CreateOrUpdateService.call(
         integration_customers: args[:integration_customers],
-        customer: result.customer,
+        customer: customer,
```

Comment:
you can align both calls to this service perhaps?

if you're passing in `customer` you can simply drop it:

```
.call(..
customer:,
...)
```

and end with `result.customer.reload` ?